### ISSUE TITLE
Question: Is it possible to override outFile command from loaded tsconfig

### ISSUE THREAD

**Issue by jkanchelov** [TypeStrong/grunt-ts#409]
Hello, guys. I bumped in some trouble loading a tsconfig and then for different builds trying to change the out ( outFile ) destination through grunt-ts. Is it possible to be done? 

I want to do something like that 
```
        debug: {
            tsconfig: './tsconfig.json',
            out: "./debug",
        },
        release: {
            tsconfig: './tsconfig.json',
            out: "./release",
        }
```

Thanks in advance for the help.

**Comment by AHaberl** [comment]
+1

**Comment by nycdotnet** [comment]
Hi - sorry for not responding sooner.  This is how this is intended to work, and I believe this is a bug.

**Comment by nycdotnet** [comment]
Should be fixed in v6.0.0-beta.19

**Comment by nycdotnet** [comment]
Thanks for the report - this was a stupid copy and paste bug and I didn't have a test for the expected behavior.
